third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -206,7 +206,7 @@ def arcs_kt_library(
             testonly = testonly,
             # Exclude any wasm-specific srcs.
             # TODO: jvm srcs will be included here. That is not what we want.
-            srcs = [src for src in srcs if not src.endswith(".wasm.kt")],
+            srcs = [src for src in srcs if src.find("_genrule_wasm") == -1],
```

Comment:
I fill like these magic suffixes should be pulled into constants, or a function like
is_src_for_platform(src, "js")